src/jwt-middleware.js

```diff
@@ -73,7 +73,7 @@ function scopeAllowsRequest(scope, request) {
     const access = parts[1];
     const readwrite = access === Constants.READWRITE;
     path = parts[0];
-    if (requestPath.startsWith(path)) {
+    if (requestPath.startsWith(path) || path.startsWith(requestPath)) {
```

Comment:
This incorrectly grants access to '/'. Instead of trying to generally accept prefixes of path it would be less error-prone to special-case instances where path starts with Constants.THINGS_PATH and requestPath is Constants.THINGS_PATH.